cpp/src/IceGrid/IceGridNode.cpp

```diff
@@ -419,9 +419,7 @@ NodeService::startImpl(int argc, char* argv[], int& status)
     //
     _timer = make_shared<IceInternal::Timer>();
 
-    //
-    // The IceGrid instance name.
-    //
+    // The IceGrid instance name. We can't use getIceProperty as we don't want to get any default values.
     string instanceName = properties->getProperty("IceGrid.InstanceName");
```

Comment:
Should we simply set the default IceGrid.InstanceName in PropertyNames.xml to empty?

There is no point setting a default value in PropertyNames.xml if it's never read.